miniapps/solvers/lor_docs.md

```diff
@@ -0,0 +1,69 @@
+# Low-order refined classes
+
+MFEM features several low-order refined (LOR) classes intended to make working with LOR discretizations and solvers more convenient.
+The interface is designed to be convenient, flexible, and powerful: solvers with "default" options can be created in only one line of code, but much more customization is available if desired.
+
+## Creating solvers with default options
+
+The most basic LOR solver can be created as a one liner. For example, the following creates a direct solver based on a serial LOR discretization:
+
+```c++
+// Given BilinearForm `a` and list of essential DOFs, create direct LOR solver
+LORSolver<UMFPackSolver> prec(a, ess_tdof_list);
+```
+This will automatically create a LOR discretization corresponding to `a`, assemble the system, and create a solver whose type is given by the template parameter.
+For example, in parallel, one could do the following to create an AMG preconditioner instead of direct solver:
+```c++
+LORSolver<HypreBoomerAMG> prec(a, ess_tdof_list);
+```
+
+### What does `LORSolver` do?
+
+In the above examples, `LORSolver<T>` does a couple of things:
+
+* Creates a LOR mesh and finite element space
+* Assembles a LOR version of the bilinear form `a`
```

Comment:
Maybe most users won't care, but at least I am interested in how the Gerritsma basis functions are handled here. If the high-order form uses the standard MFEM basis, can you do anything with LOR? If so, how does it work?

Reply:
This is a good point (which, incidentally, @v-dobrev also brought up today).

One option is to warn the user if the "wrong" basis type is being used. Another option is to apply a change of basis operation in the solver action the same way the DOF permutation is currently applied. I'll have to look more into both of these options to figure out the best way to go.

Reply:
I think the best thing to do is to automatically apply a change of basis, but doing that properly and efficiently is somewhat involves, and will probably happen in a subsequent PR.

Reply:
It's fine for that to be a follow-on PR, but I think the documentation should be very clear about when a user can expect good performance from an LOR solver (IMO it's not quite clear enough right now).

Reply:
I updated the documentation explaining which bases are spectrally equivalent, and I added code that checks this at runtime and emits a warning when non-spectrally equivalent bases are used.